test/utils/expose_keptn_bridge.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
```

Comment:
Can we please rename this file in `check_bridge_availability.sh` or something similar?